app/decorators/manageiq/providers/automation_manager/inventory_root_group_decorator.rb

```diff
@@ -1,5 +1,5 @@
-module ManageIQ::Providers::AutomationManager
-  class InventoryRootGroupDecorator < MiqDecorator
+module ManageIQ::Providers
```

Comment:
@lgalis the same change will be needed in `app/decorators/manageiq/providers/automation_manager/inventory_group_decorator.rb` as well.